src/Apple/Provider.php

```diff
@@ -196,15 +196,17 @@ public function user()
      */
     protected function mapUserToObject(array $user)
     {
+        $fullName = '';
+
         if (request()->filled('user')) {
             $userRequest = json_decode(request('user'), true);
 
             if (array_key_exists('name', $userRequest)) {
                 $user['name'] = $userRequest['name'];
                 $fullName = trim(
-                    ($user['name']['firstName'] ?? '')
```

Comment:
null coalesce is in 7.0

Reply:
Same as other comments @atymic, all those providers are supposed to work with `PHP 5.6`...

Introducing PHP 7 or 7.1 specific syntax breaks this compatibility.

Reply:
Ok, feel free to drop < 7 then. We really need to drop < 7.2 some time soon as well.